### ISSUE TITLE
Tickets assigned to account without full name set didn't print

### ISSUE THREAD

**Issue by sm6xmm** [EuroPython/epcon#750]
Tickets that were assigned to an account without full name set didn't get printed, most likely they were assigned before the account existed, and so the account may not have been active when we printed the tickets.

Perhaps we want to ask for full name also when assigning tickets.

**Comment by artcz** [comment]
This is probably no longer relevant because the assignment flow no longer allows for "assign to a non existent user and send them invite to create an account" flow.

@sm6xmm @hypha did you notice anything like that on the 2019 badges? If not we can probably close this ticket.

**Comment by sm6xmm** [comment]
> This is probably no longer relevant because the assignment flow no longer allows for "assign to a non existent user and send them invite to create an account" flow.
> 
> @sm6xmm @hypha did you notice anything like that on the 2019 badges? If not we can probably close this ticket.

No, unless the wrong name/number thing we saw is somehow related.

**Comment by sm6xmm** [comment]
Also, we need to discuss whether, and in that case how, we should enable that flow again. It will be impractical for corporate buyers if they have to come back and reassign tickets because some people don't yet have accounts (as we saw this year with multiple tickets on one user).

**Comment by hypha** [comment]
No. Haven't noticed this issue. And had a look at the wrong badges. Fairly certain it is not related to this.
